lib/3scale/backend/stats/cleaner.rb

```diff
@@ -245,6 +252,16 @@ def delete_stats_keys_with_val_0(redis_conn, log_deleted_keys)
               sleep(SLEEP_BETWEEN_SCANS)
             end
           end
+
+          def with_retries
+            begin
+              retries ||= 0
```

Comment:
`retries = 0` could be used before `begin` and you wouldn't need to check again nor initialize with `||=`.

Reply:
Fixed.